libsolidity/inlineasm/AsmCodeGen.cpp

```diff
@@ -157,5 +157,5 @@ void assembly::CodeGenerator::assemble(
 		false,
 		_identifierAccess,
 		_useNamedLabelsForFunctions
-	)(_parsedData);
+	).initReferencesCounter(_parsedData)(_parsedData);
```

Comment:
Ah I'm sorry, I did not see that `CodeTransform` does not receive the AST in the constructor.
Another option might be to just do that in `::operator()(Block`, but it might be better this way.

Reply:
Yeah, it is better to pass to the constructor